girder/utility/install.py

```diff
@@ -148,6 +155,9 @@ def main():
     plugin.set_defaults(func=install_plugin)
     plugin.add_argument('-f', '--force', action='store_true',
                         help='Overwrite plugins if they already exist.')
+    plugin.add_argument('-e', '--editable', action='store_true',
+                        help='Install by symlinking to the plugin directory.')
```

Comment:
I appreciate that this is consistent with the pip install flag, but to me it seems much clearer to just use `-s, --symlink` as the name of this option. Do others feel strongly about this?